language/Pattern-Matching.md

```diff
@@ -176,6 +176,17 @@ compare _ _ = EQ
 
 (The name `otherwise` is a synonym for `true` commonly used in guards.)
 
+You can also use commas to add multiple expressions in a guard:
+
+```purescript
+positiveLessThanFive :: Maybe Int -> Boolean
+positiveLessThanFive mInt
+  | Just x <- mInt
+  , x > 0
+  , x < 5 = true
+  | otherwise = false
+```
+
```

Comment:
Should this be moved into the `Pattern Guards` section below? Commas are only necessary because of pattern guards.

Reply:
oh yeah, i'll move it down